web/components.d.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable */
+/* prettier-ignore */
+// @ts-nocheck
+// Generated by unplugin-vue-components
+// Read more: https://github.com/vuejs/core/pull/3399
+export {}
```

Comment:
`unplugin-vue-components` contains vite plugins for various vue packages, including Vuetify 2. It's used because the [official Vuetify Vite plugin loader](https://github.com/unplugin/unplugin-vue-components) only supports Vue 3. 

Upon coming back to this, I think this `components.d.ts` file is not intended to be committed to version control. It's autogenerated whenever `vite` is run, and the unplugin-vue-components repo has it in [their .gitignore](https://github.com/unplugin/unplugin-vue-components/blob/main/.gitignore#L5), so I just removed it and added it to `web/.gitignore` https://github.com/dandi/dandi-archive/pull/1725/commits/6998caa63445437d4a05a7cd380e8ae3c6d46983. 

This is more or less just a stopgap measure to enable using Vuetify with Vite while we're still on Vue 2. My next step after this PR is to do the upgrade to Vue 3, after which we will no longer need this plugin library anymore.